### PR TITLE
refresh descent_two_isogeny.pyx

### PR DESCRIPTION
**Pull request by fchapoton**
mostly pep8 formatting for the code

also removing 3 unused variables found by cython-lint

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/schemes/elliptic_curves/descent_two_isogeny.pyx [hunk 1]
```diff
@@ -159,17 +164,21 @@ cdef int lemma6(mpz_t a, mpz_t b, mpz_t c, mpz_t d, mpz_t e,
...
-        if lambd >= 2*nu: result = 0 # undecided
+    if mpz_sgn(g_prime_of_x) == 0:
+        if lambd >= 2*nu:
+            result = 0  # undecided
     else:
         mu = valuation(g_prime_of_x, p)
-        if lambd > 2*mu: result = +1 # soluble
-        elif lambd >= 2*nu and mu >= nu: result = 0 # undecided
+        if lambd > 2*mu:
+            result = +1  # soluble
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/descent_two_isogeny.pyx]: Simply `result = 1`

File: src/sage/schemes/elliptic_curves/descent_two_isogeny.pyx [hunk 2]
```diff
@@ -209,28 +218,33 @@ cdef int lemma7(mpz_t a, mpz_t b, mpz_t c, mpz_t d, mpz_t e,
...
+        if lambd >= 2*nu:
+            result = 0  # undecided
         elif lambd == 2*nu-2 and g_of_x_odd_part_mod_4==1:
-            result = 0 # undecided
+            result = 0  # undecided
     else:
         mu = valuation(g_prime_of_x, p)
-        if lambd > 2*mu: result = +1 # soluble
+        if lambd > 2*mu:
+            result = +1  # soluble
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/descent_two_isogeny.pyx]: same here and other places

File: src/sage/schemes/elliptic_curves/descent_two_isogeny.pyx [hunk 3]
```diff
@@ -928,10 +1001,15 @@ def test_els(a,b,c,d,e):
         ....:         except ValueError:
         ....:             continue
     """
-    cdef Integer A,B,C,D,E,Delta
-    A=Integer(a); B=Integer(b); C=Integer(c); D=Integer(d); E=Integer(e)
+    cdef Integer A,B,C,D,E
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/descent_two_isogeny.pyx]: `cdef Integer A, B, C, D, E` ?

**Review by dcoudert — CHANGES_REQUESTED** [review]
few remarks.

**Comment by fchapoton** [comment]
ok, c'est fait. merci

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**